app/services/reports/region_cache_warmer.rb

```diff
@@ -32,19 +32,21 @@ def call
     end
 
     def warm_region_cache(region)
-      RequestStore.store[:bust_cache] = true
+      Time.use_zone(Period::ANALYTICS_TIME_ZONE) do
```

Comment:
👍

Reply:
Extracted to https://github.com/simpledotorg/simple-server/pull/2569